### PR TITLE
Port targetconfig changes to v0

### PR DESCRIPTION
**Pull request by cait1inhennessy**
Attempting to port https://github.com/Microsoft/pxt/pull/3526 to v0.

Running "pxt serve --cloud" in pxt-microbit now gives the following error:

capture

### CODE REVIEW

File: localtypings/pxtarget.d.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ declare namespace pxt {
         languages?: string[];
         notifications?: Map<Notification>; // domain specific notifications to show users on load
         windowsStoreLink?: string;
+        pineapple?: string;
```
Comment (pelikhan) [inline localtypings/pxtarget.d.ts]: is this new?

File: webapp/src/notificationbanner.tsx [hunk 2]
```diff
@@ -101,10 +101,7 @@ export class NotificationBanner extends data.Component<ISettingsProps, {}> {
 
         if (showWindowsStoreBanner) {
             return (
-                <GenericBanner parent={this.props.parent} delayTime={10000} displayTime={45000} sleepTime={604800}>
-                    <sui.Link class="link" target="_blank" ariaLabel={lf("View app in the Windows store")} href={targetConfig.windowsStoreLink} onClick={() => pxt.tickEvent("banner.linkClicked")}>
-                        <img className="bannerIcon" src={Util.pathJoin(pxt.webConfig.commitCdnUrl, `images/windowsstorebag.png`)}></img>
-                    </sui.Link>
+                <GenericBanner parent={this.props.parent} delayTime={1000}>
```
Comment (pelikhan) [inline webapp/src/notificationbanner.tsx]: @cait1inhennessy please check this line

**Comment by pelikhan** [comment]
I've merged missing changes. Please check that all pineapples are gone.

**Comment by cait1inhennessy** [comment]
Should be good to go now.

**Pull request merged**